setup.py

```diff
@@ -12,10 +12,26 @@
 with open(os.path.join(repository_dir, SOURCE_DIR, '_version.py')) as fh:
     exec(fh.read())
 
+# Use readme needed as long description in PyPI
+with open(os.path.join(repository_dir, 'README.md')) as fh:
```

Comment:
Similar comment to the first one, this will likely be blatted / re-done with cookiecuttering over this repo.